sagui.config.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Sagui configuration object
+ */
+module.exports = {
+  /**
+   * Different application entry-points
+   * Each page is a combination of a JavaScript file and a HTML file
+   *
+   * Example: 'index' -> 'index.html' and 'index.js'
+   */
+  pages: ['index', 'source', 'target'],
```

Comment:
Yes, absolutely, I just didn't understand how to use the Library feature. I didn't know what name should I put in the `library` config and what entry point would be configured. Also I wasn't sure how to specify what do load since I knew that nested folders didn't work yet.

If you feel like it, just push the changes to the branch however you see fit.